### PR TITLE
Fixes stomach for synths with oversized quirk

### LINKED ISSUE #24506: Oversized quirk replaces synthetic humanoid's bio-reactor
## Round ID:
9713

## Testmerges:
[#24068: 'URL Clickabillity. OOC Notes/Flavor Text/Species lore' by AnywayFarus at commit 2da0af6ca6](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24068)
[#24264: 'Disables pipe gas visuals for now' by vinylspiders at commit 849b6ed3bf](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24264)
[#23604: '[READY] [MODULAR] Blueshift wakes up after taking "Worst Nap Ever" [MDB IGNORE]' by W4rd3nn at commit d37a051f7c](https://github.com/Skyrat-SS13/Skyrat-tg/pull/23604)
[#24056: 'Icecat camp but epic!! | Visual updates to stuff that made the before mentioned camp look awful | Hazardously concentrated amounts of :3' by Paxilmaniac at commit cfd7abe6f6](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24056)
[#24058: 'Cargo Engineering Content Part 1 | My little machines that go wwwhrhhhr | Die of CO2 poisoning' by Paxilmaniac at commit 0960012ea9](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24058)

## Reproduction:

1. Select synthetic humanoid
2. Take the oversized quirk
3. Enter the round
4. Your bioreactor is now replaced by 'huge gut' removing the ability to charge until replaced by a bio reactor.

While not the largest of issues - in terms of gameplay and coding - it can be rather annoying to bother robotics every shift to make sure you do not starve.

### PR DESCRIPTION
**Pull request by Plum-but-gayer**
## About The Pull Request

Adds a new oversized synth stomach and gives to synth characters with the oversized quirk. i copied the stats over from the other big stomach, and made a placeholder sprite (resized synth stomach. it's bigger.) 

Fixes [#24506](https://github.com/Skyrat-SS13/Skyrat-tg/issues/24506)

## How This Contributes To The Skyrat Roleplay Experience
Big Robot Good.

## Proof of Testing

Tested on runtime station, ran around, charged my battery, spawned in as an organic/synth, did surgery to remove the organ.

<details>
<summary>Screenshots/Videos</summary>
  
image

image

image

</details>

## Changelog

:cl:
add: Added a new oversized synthetic stomach, for oversized synths.
fix: Fixed  oversized synths spawning with organic stomachs.
image: added a sprite for the oversized synth stomach. It's a lil jank, sorry!
/:cl:

### CODE REVIEW

**Comment by Plum-but-gayer** [comment]
do I get good boy points for this also idk how gbp works

**Comment by ChickyRooster** [comment]
McThankies

File: modular_skyrat/modules/organs/code/stomach.dm [hunk 1]
```diff
@@ -5,3 +5,13 @@
...
 	metabolism_efficiency = 0.07
+
+/obj/item/organ/internal/stomach/synth/oversized
+	name = "huge synthetic bio-reactor"
+	desc = "Typically found in huge synthetics, this monstrous engine has developed to be highly efficient, made to get an enormous amount of power to an enormous machine."
+	icon = 'modular_skyrat/modules/organs/icons/stomach.dmi'
+	icon_state = "stomach_big_synth" //ugly placeholder sorry im not an artist hehe
+	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD
+	metabolism_efficiency = 0.07
+
```
Comment (vinylspiders) [inline modular_skyrat/modules/organs/code/stomach.dm:16]:
```suggestion
```

File: modular_skyrat/modules/organs/code/stomach.dm [hunk 2]
```diff
@@ -5,3 +5,12 @@
 	icon_state = "stomach_big"
 	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD
 	metabolism_efficiency = 0.07
+
+/obj/item/organ/internal/stomach/synth/oversized
+	name = "huge synthetic bio-reactor"
+	desc = "Typically found in huge synthetics, this monstrous engine has developed to be highly efficient, made to get an enormous amount of power to an enormous machine."
```
Comment (vinylspiders) [inline modular_skyrat/modules/organs/code/stomach.dm]:
```suggestion
	desc = "Typically found in huge synthetics, this monstrous engine has been developed to be highly efficient, made to provide an enormous amount of power to an enormous machine."
```

**Review by vinylspiders — APPROVED** [review]
Looks fine to me! and you will get gbp yes, check the gbp update at the bottom of the PR once it is merged and you can see how much you got.

You should get 3 for this since it's a fix.

**Pull request merged**